daisy_workflows/image_import/import_image.sh

```diff
@@ -48,24 +54,50 @@ function resizeDisk() {
   fi
 }
 
-# Mount GCS bucket containing the disk image.
-mkdir -p /gcs/${SOURCEBUCKET}
-gcsfuse --implicit-dirs ${SOURCEBUCKET} /gcs/${SOURCEBUCKET}
+function copyImageToScratchDisk() {
+  local sourceSizeGigabytes="$((1 + (SOURCE_SIZE_BYTES - 1) / BYTES_1GB))"
+  local scratchDiskSizeGigabytes=sourceSizeGigabytes
```

Comment:
also unnecessary second variable

Reply:
I was hoping to add clarity of the size of the image versus the size of the disk, but now that you point it out, I agree that it'd be clearer to consolidate it!